frontend/src/components/elements/DataFrame/DataFrameCell.tsx

```diff
@@ -26,8 +27,8 @@ export interface DataFrameCellProps {
   /** The cell's row index in the DataFrame */
   rowIndex: number
 
-  /** The cell's css class name */
-  className: string
+  /** The cell's component to render */
+  as: ComponentType
```

Comment:
Can we use something other than `as`? This can become confusing as TypeScript also has the `as` keyword.